### DISCUSSION TITLE
applying ww3_grid.inp file to a global bathemetry set

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#960] [General]
I have a ww3_grid.inp file that works perfectly with output  boundary conditiions which is attached
[ww3_grid.inp.txt](https://github.com/NOAA-EMC/WW3/files/11074938/ww3_grid.inp.txt).

This is all good but how to I apply this to other bathemetry condtions?   For instance I have the following global bathemetry data files that I want this ww3_grid.inp to work with which are attached here:
[global_1p0_mask_new.dat.txt](https://github.com/NOAA-EMC/WW3/files/11074948/global_1p0_mask_new.dat.txt)
[global_1p0_obst_new.dat.txt](https://github.com/NOAA-EMC/WW3/files/11074950/global_1p0_obst_new.dat.txt)
[global_1p0_bathy_new.dat.txt](https://github.com/NOAA-EMC/WW3/files/11074952/global_1p0_bathy_new.dat.txt)

When I run the original ww3_grid.inp that is attached the file runs perfectly with output boundary conditions shown.  However when I try to modify this file to process the global bathemetry files attached I get an error?   How do I modify the ww3_grid.inp file to successfully process the above global bathemetry files with the output boundary conditions?   The bathemetry filses cover longitude 0 to 360 degrees longitude and -78 to 80 degrees latitude?  The error I get is the following:

  Output boundary points :
 --------------------------------------------------

 *** WAVEWATCH III ERROR IN W3GRID :
     ERROR IN READING FROM INPUT FILE
     IOSTAT =    0

**Comment by CarstenHansen** [comment]
This question has been copied into #959 and discussed there.
